tectonic/internals.h

```diff
@@ -136,7 +136,7 @@ void set_input_file_encoding(UFILE *f, integer mode, integer encodingData);
 void u_close(UFILE *f);
 int u_open_in(UFILE **f, integer filefmt, const char* fopen_mode, integer mode, integer encodingData);
 int get_uni_c(UFILE* f);
-int input_line(UFILE* f);
+boolean input_line(UFILE* f);
```

Comment:
The `boolean` typedef has been [removed](https://github.com/tectonic-typesetting/tectonic/commit/f0ec7c6f994b02274d9ee1cd5a08d9cb10a3dbb5). You should change the `int` to `bool` instead.